applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -804,22 +820,28 @@ message_count_prompts(New, Saved) ->
                             kapps_call_command:audio_macro_prompts().
 message_prompt([H|_]=Messages, Message, Count, #mailbox{timezone=Timezone
                                                        ,skip_envelope='false'
+                                                       ,allow_ff_rw=AllowFfRw
                                                        }) ->
     [{'prompt', <<"vm-message_number">>}
     ,{'say', kz_term:to_binary(Count - length(Messages) + 1), <<"number">>}
-    ,{'play', Message}
+    ,play_prompt(Message, AllowFfRw)
     ,{'prompt', <<"vm-received">>}
     ,{'say',  get_unix_epoch(kz_json:get_integer_value(<<"timestamp">>, H), Timezone), <<"current_date_time">>}
     ,{'prompt', <<"vm-message_menu">>}
     ];
-message_prompt(Messages, Message, Count, #mailbox{skip_envelope='true'}) ->
+message_prompt(Messages, Message, Count, #mailbox{allow_ff_rw=AllowFfRw
+                                                 ,skip_envelope='true'}) ->
     lager:debug("mailbox is set to skip playing message envelope"),
     [{'prompt', <<"vm-message_number">>}
     ,{'say', kz_term:to_binary(Count - length(Messages) + 1), <<"number">>}
-    ,{'play', Message}
+    ,play_prompt(Message, AllowFfRw)
     ,{'prompt', <<"vm-message_menu">>}
     ].
 
+play_prompt(Message, true=_AllowFfRw) ->
+    {'play', Message, ?VM_MESSAGE_TERMINATORS};
```

Comment:
If I customize `keys` in the callflow vm system_config to something other than 5 and 8, this will terminate playback instead of ff/rw.

Reply:
Also please tickie all atoms: `true` should be `'true'`.

Reply:
@jamesaimonetti latest code has been updated for this, please take a look :)